apps/site/assets/ts/stop/components/LocationBlock.tsx

```diff
@@ -53,15 +48,16 @@ const streetViewLink = (stop: Stop): string =>
 const LocationBlock = ({
   routes,
   stop,
-  encoder,
   streetViewUrl
 }: Props): ReactElement<HTMLElement> => (
   <div className="m-stop-page__location-block">
     {addressOrMunicipality(stop)}
     <div className="m-stop-page__location-links">
       <div className="m-stop-page__location-link">
         <a
-          href={directionLink(stop, encoder)}
+          href={formatMapLink(
+            stop["station?"] ? `${stop.name}-station` : stop.address
```

Comment:
When `stop.address` is `null` this ends up taking the user back to the stop page. BUT I just realized that the link seems to work fine with latitude/longitude - I just tried `/trip-planner/to/42.255009,-71.006352` and it worked great. So let's use `latLngString(stop)` here in place of null address (or maybe in place of address altogether? I haven't yet found a not-station with a valid address though...)